lib/core/src/lib/snackbar-content/snackbar-content.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {SnackbarContentComponent} from '@alfresco/adf-core';
+import {MatIcon, MatIconModule} from '@angular/material/icon';
+import {MAT_SNACK_BAR_DATA, MatSnackBarModule, MatSnackBarRef} from '@angular/material/snack-bar';
+import {MatButton, MatButtonModule} from '@angular/material/button';
+import {By} from '@angular/platform-browser';
+
+describe('SnackbarContentComponent', () => {
+    let component: SnackbarContentComponent;
+    let fixture: ComponentFixture<SnackbarContentComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [SnackbarContentComponent],
+            imports: [
+                MatIconModule,
+                MatSnackBarModule,
+                MatButtonModule
+            ],
+            providers: [{
+                provide: MatSnackBarRef,
+                useValue: {
+                    dismissWithAction() {
+                    }
+                }
+            }, {
+                provide: MAT_SNACK_BAR_DATA,
+                useValue: {}
+            }]
+        })
+            .compileComponents();
+
+        fixture = TestBed.createComponent(SnackbarContentComponent);
+        component = fixture.componentInstance;
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should display message if message in data is set', () => {
+        // given
```

Comment:
no need using comments in the tests as the flow is already obvious, we don't use code comments

Reply:
done